README.md

```diff
@@ -11,89 +11,227 @@ Library used only for implementors of custom listeners for ReportPortal.
 
 ## Installation
 The latest version is available on npm:
+```cmd
+npm install reportportal-client
+```
+    
+
+
+## Example
 
-    npm install reportportal-client
+```javascript
+let RPClient = require('reportportal-client');
 
+let rpClient = new RPClient({
+    token: "00000000-0000-0000-0000-000000000000",
+    endpoint: "http://your-instance.com:8080/api/v1",
+    launch: "LAUNCH_NAME",
+    project: "PROJECT_NAME"
+});
+
+rpClient.checkConnect().then((responce) => {
+    console.log('You have successfully connected to the server.');
+    console.log(`You are using an account: ${responce.full_name}`);
+}, (error) => {
+    console.log('Error connection to server');
+    console.dir(error);
+});
+```
 
-## Usage
+## Settings
+When creating a client instance, you need to specify the following parameters:
+
+Parameter | Description
+--------- | -----------
+token     | user's token Report Portal from which you want to send requests. It can be found on the profile page of this user.
+endpoint  | URL of your server. For example, if you visit the page at 'https://server:8080/ui', then endpoint will be equal to 'https://server:8080/api/v1'.
+launch    | Name of lunch at creation.
+project   | The name of the project in which the lunches will be created.
+
+## Api
+Each method (except checkConnect) returns an object of a specific format:
+```javascript
+{
+    tempId: '4ds43fs', // generated by the client id for further work with the created item
+    promise: Promise // An object indicating the completion of an operation
+}
+```
+The client works synchronously, so to send the following requests it is not necessary to wait for the end of previous ones.
```

Comment:
so it is not necessary to wait for the end of the previous requests to send following ones.